modules/react/collection/stories/Collection.stories.mdx

```diff
@@ -0,0 +1,130 @@
+the same was what's provided to the model. By default, providing items will enable virtualization.
+This example adds a `maxHeight` to ensure overflow. Virtualization uses absolute positioning of each
+item, which could cause problems for popup menus. If your item count is low, pass
+`shouldVirtualize={false}` to disable virtualization.
+
+<ExampleCodeBlock code={DynamicItems} />
+
+### List Model
+
+The List model contains the the state and events necessary to track items, selection, and a cursor.
+Various hooks can be used for a List model to create common behaviors associated with lists, such as
+navigating a list with a keyboard, selection (single and multiple), and virtualization.
+
+A list also has a "cursor". A cursor is often represented by focus, but it is not always a 1:1
+mapping. Think of the cursor as the focus item within the list. If the list has browser focus, the
+cursor will map to browser focus. Behaviors such as `useListRovingFocus` will map the cursor to the
+active tab stop of the list. For more information, see
+[Roving Tabindex](https://w3c.github.io/aria-practices/#kbd_roving_tabindex). `useListRovingFocus`
+adds keyboard events that map to navigation events. A [Navigation Manager](#navigation-manager) is
+used to map new cursor ids to these events. The `ListModel` takes an optional `navigation`
+configuration to change the default navigation behavior. The default navigation manager is a
+[wrappingNavigationManager](#wrappingnavigationmanager) meaning the cursor will wrap around the
+beginning and the ends. The cursor also provides a [navigationManager](#navigationmanager) that does
+not wrap. This is the default navigation for grids.
+
+The cursor also adds the concept of `orientation` which defaults to `'vertical'`. A Tab list is an
+example of a `'horizontal'` list.
+
+### Grid
+
+A Cursor List can become a grid if a `columnCount` is provided. The array of items is still flat,
+but navigational controls are now in two dimensions instead of one. A one dimension list is
+bidirectionally aware, but a grid will always
+
+#### Navigation Manager
+
+#### Roving Tabindex
+
+The list system also includes a cursor that extends the list. A cursor is mostly used for focusing
+items. The [roving tabindex](https://w3c.github.io/aria-practices/#kbd_roving_tabindex) is a
+well-supported way to accomplish accessibility requirements for focusing items within a list. This
+example shows how to use `useListRovingFocus`. You can either use the tab key for focus on an item
+or click on an item and then use the up/down keys to navigation the list. By default, the list is
+set to wrap navigation using the `wrappingNavigationManager`. Only a single item in the list is a
+focus stop that "roves" as the up/down arrows are pressed.
+
+**Note:** This example doesn't meet accessibility requirements. The list will have to have some type
+of context. Like "navigation list" or "menu list".
+
+<!--ExampleCodeBlock code={RovingFocus} /-->
```

Comment:
did you still want this commented out?

Reply:
I didn't revisit this after introducing a `ListBox` which takes care of most of this. Now I'm a bit torn on how to document the collection system...

Most cases are covered by `ListBox`, but not all. Originally I was showcasing how all the low-level models and hooks could be used together to create higher-level components. Now I've introduced a few abstractions that cover the common cases I'm outlining and `ListBox` should be used when possible. So... Do I stick with low-level or use the highest abstraction possible?